.pylintrc

```diff
@@ -483,7 +483,7 @@ ignore-comments=yes
 ignore-docstrings=yes
 
 # Ignore imports when computing similarities.
-ignore-imports=no
+ignore-imports=yes
```

Comment:
Enable this as it compares imports to each other and that obviously then causes an error when linting.